decredPoS/decredPoS_script.md

```diff
@@ -1,43 +1,40 @@
+After 256 blocks (or about 20 hours), immature tickets become live and are added to the ticket pool. How long each ticket has to wait in the  pool before being drawn is entirely up to chance. Due to the design of 
+the voting system, the average time to vote is 28 days and the maximum waiting time is 142 days.
 
-How long an individual ticket will wait in the pool before being drawn is entirely up to chance, but current data shows the average waiting time is 30 days.
-
-Once a stakeholder's ticket is drawn, their voting ballot must be broadcast to the network from either their own *self-hosted* **voting wallet** or a designated **voting service provider** for inclusion the next block. If a ballot is not broadcast quickly, its votes may never be cast. This occurs in less than 2% of cases and is caused by bad internet connections and network races.
+Each block randomly draws 5 tickets from the ticket pool. Once a ticket is called, the holder's signed voting ballot must be broadcast to the network from either a self-hosted voting wallet or a designated voting service provider for inclusion in the next block. If the ballot is not broadcast quickly enough, it will miss its chance to vote. This happens rarely, as most solo voters and voting service providers maintain several wallets. Missed tickets can be revoked after 256 blocks but do not get a reward.
 
 A ticketholder's ballot will contain one **block vote** and zero to many **consensus votes** to be recorded *on-chain*.
 
 Block votes allow stakeholders to keep proof-of-work miners in check. The 5 block votes contained in each new block decide whether to accept or reject the previous one. If stakeholders decide to reject the block, its pending transactions are returned to the mempool and the miner is stripped of their block reward.
 
-Consensus votes allow stakeholders to prevent unwanted hard-forks through a two stage process where stakeholders vote for or against changes to Decred's consensus rules.
+Consensus votes allow stakeholders to change Decred's consensus rules through a two stage process where stakeholders vote for or against the proposed change.
 
-Before the consensus voting process can begin, a majority of nodes must upgrade to the newly released software version containing the proposed rule change. The update threshold is met when 95% of the 1000 most recent blocks and 75% of the proof of stake votes within *one* **stake version interval**, 2016 blocks (or about 1 week), have the latest version. 
+Before the consensus voting process can begin, nodes must update to new software that has a latent copy of the proposed new rules. If the proposed rule change is approved, these will be automatically activated. The update threshold is met when 95% of the 1000 most recent blocks were mined by updated nodes and 75% of the proof of stake votes within *one* **stake version interval**, 2016 blocks (or about 1 week), have the latest version. 
```

Comment:
Instead of "latent copy" I would go with "inactive code" here. Later on, replace "these" with "the sleeping code"